### DISCUSSION TITLE
Multiple suggestions in a row ?

### DISCUSSION THREAD

**Discussion by ajnart** [c4spar/cliffy#313] [Q&A] [answered]
First I would like to thank you, @c4spar for putting so much time into this library, it has been a joy to use while learning deno and typescript. 

I have a question, **is it possible to have multiple suggestions autofill in one prompt ?**
I'll explain myself:  
I am trying to create a small parser that takes in a string with logic blocks that have distinct names (I.e. ``{master}{second}``) 
I would really love to be able to autocomplete multiple suggestions within the same prompt, but I haven't been able to find any help about this particular use of case.

**Comment by c4spar** [comment, marked as answer]
Hi @ajnart,

this is currently not possible. You could use the List prompt which takes multiple values which are completed indiviually but i think it's not what you want. I can investigate how easy it is to implement something like this.

**Reply by ajnart** [reply]
> Hi @ajnart,
> 
> this is currently not possible. You could use the List prompt which takes multiple values which are completed indiviually but i think it's not what you want. I can investigate how easy it is to implement something like this.

Thank you for the quick reply, I didn't see it. It would be really nice of you to investigate how easily you could implement that feature.

**Reply by gavriguy** [reply]
I also have the similar request - I want to create a prompt that auto suggest tags / users that i have stored.
